Quantum-Control-Applications/Superconducting/Two-Fixed-Coupled-Transmons/README.md

```diff
@@ -0,0 +1,97 @@
+# Two Fixed Frequency Coupled Transmon Superconducting Qubit
+
+<img align="right" src="Two Flux Tunable Transmon Setup.PNG" alt="drawing" width="400"/>
+
+## Experimental setup and context
+
+This repository includes a collection of scripts for performing various experiments on two fixed-frequency transmons,
+encompassing qubit calibration, cross-resonance (CR) gate tuning, and CNOT gate formation.
+Each transmon is equipped with individual qubit drive lines and a shared readout transmission line.
```

Comment:
Each transmon is equipped with individual qubit drive line and a shared readout transmission line.

Reply:
fixed the grammar.